index.html

```diff
@@ -44,13 +44,13 @@
                   <!--Add buttons to initiate auth sequence and sign out-->
                   <form action="#">
                         <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-                          <input class="mdl-textfield__input" type="text" id="sample3">
+                          <input class="mdl-textfield__input" type="text" id="sample search-input">
```

Comment:
This id is '#sample search-input', could you remove 'sample ' and change the ```for``` attribute on the label below to match this inputs new id?